src/ui/public/courier/fetch/request/__tests__/segmented.js

```diff
@@ -41,7 +53,7 @@ describe('SegmentedRequestProvider', () => {
 
   function mockSource() {
     return {
-      get: sinon.stub().returns(mockIndexPattern())
+      get: sinon.stub().returns(mockIndexPattern()),
```

Comment:
Why the extra comma here?

Reply:
I added a line below it at one point and deleted it later